examples/basic_pos_tagger.py

```diff
@@ -24,8 +69,38 @@ def main(
             >> ReLu(width, width)
             >> Softmax(17, width)
         )
```

Comment:
I guess it's a matter of personal preference, but I find it more easily understandable if you add `nO=` and `nI=` bits to the layer definitions. Once you know what these abbreviations mean, I find it helps understanding the flow better.

Reply:
Ah, so you mean `Softmax(nO=17, nI=width)`? If we do this, I think we should adopt this consistently across the examples. @honnibal What do you think?

Reply:
Yeah it's probably a bit important: our layers have the args swapped compared to PyTorch, they go `(input_size, output_size)`. We don't want to do that because we more often allow the input size to be omitted, but it does mean the strictly positional args are probably not great in initial examples :(.